docs/howto/operate/new-cluster/aws.md

```diff
@@ -204,6 +197,50 @@ have least amount of permissions possible.
    The `aws.key` file is defined _relative_ to the location of the `cluster.yaml` file.
    ```
 
+## Scaling up a nodegroup in a cluster
+
+`eksctl` creates nodepools that are mostly immutable, except for autoscaling properties -
+minimum and maximum number of nodes. In certain cases, it might be helpful to 'scale up'
+a nodegroup in a cluster before an event, to test cloud provider quotas or to make user
+server startup faster.
+
+1. Open the appropriate `.jsonnet` file for the cluster in question, and set the
+   `minNodes` property of `notebookNodes` to the appropriate number you want.
+
+   ```{warning}
+   It is currently unclear if *lowering* the `minNodes` property just allows
+   the autoscaler to reclaim nodes, or if it actively destroys nodes at time
+   of application! If it actively destroys nodes, it is unclear if it does
+   so regardless of user pods running in these nodes! Until this can be
+   determined, please do scale-downs only when there are no users on
+   the nodes.
+   ```
```

Comment:
@yuvipanda, I was able to scale down warmed clusters setting min nodes back to zero and that does not seem to be a destructive action. People kept working as normal and nodes were reclaimed when they were emptied.